### ISSUE TITLE
Line graph and scatter plot fail with particular date vs. number data

### ISSUE THREAD

**Issue by austinpray** [FormidableLabs/victory#138]
## Issue

I have a line graph and a scatter plot I want to plot where the x axis is a series of dates and the y is a series of numbers.

## Sample data

The following function generates data that intermittently fails to render. It almost behaves like a race condition.

```js
import {random} from "lodash";

const data = {
thirtyDays: (function () {
    const startDate = new Date(2015, 12, 28);
    const result = [];
    for (let x = 0; x < 30; x++) {
      const newDate = new Date();
      newDate.setDate(startDate.getDate() - x);
      result.push({x: newDate, y: random(100, 200)});
    }
    return result;
  })()
}
```

Sample output would look like:

```json
[
   {
      "x":"2015-12-29T03:25:25.656Z",
      "y":135
   },
   {
      "x":"2015-12-28T03:25:25.657Z",
      "y":170
   },
   {
      "x":"2015-12-27T03:25:25.657Z",
      "y":113
   },
   {
      "x":"2015-12-26T03:25:25.657Z",
      "y":158
   },
   {
      "x":"2015-12-25T03:25:25.657Z",
      "y":169
   },
   {
      "x":"2015-12-24T03:25:25.657Z",
      "y":189
   },
   {
      "x":"2015-12-23T03:25:25.657Z",
      "y":151
   },
   {
      "x":"2015-12-22T03:25:25.657Z",
      "y":124
   },
   {
      "x":"2015-12-21T03:25:25.657Z",
      "y":140
   },
   {
      "x":"2015-12-20T03:25:25.657Z",
      "y":173
   },
   {
      "x":"2015-12-19T03:25:25.657Z",
      "y":175
   },
   {
      "x":"2015-12-18T03:25:25.657Z",
      "y":130
   },
   {
      "x":"2015-12-17T03:25:25.657Z",
      "y":166
   },
   {
      "x":"2015-12-16T03:25:25.657Z",
      "y":110
   },
   {
      "x":"2015-12-15T03:25:25.657Z",
      "y":115
   },
   {
      "x":"2015-12-14T03:25:25.657Z",
      "y":153
   },
   {
      "x":"2015-12-13T03:25:25.657Z",
      "y":182
   },
   {
      "x":"2015-12-12T03:25:25.657Z",
      "y":144
   },
   {
      "x":"2015-12-11T03:25:25.657Z",
      "y":184
   },
   {
      "x":"2015-12-10T03:25:25.657Z",
      "y":118
   },
   {
      "x":"2015-12-09T03:25:25.657Z",
      "y":188
   },
   {
      "x":"2015-12-08T03:25:25.657Z",
      "y":161
   },
   {
      "x":"2015-12-07T03:25:25.657Z",
      "y":158
   },
   {
      "x":"2015-12-06T03:25:25.657Z",
      "y":132
   },
   {
      "x":"2015-12-05T03:25:25.657Z",
      "y":109
   },
   {
      "x":"2015-12-04T03:25:25.657Z",
      "y":152
   },
   {
      "x":"2015-12-03T03:25:25.657Z",
      "y":197
   },
   {
      "x":"2015-12-02T03:25:25.657Z",
      "y":143
   },
   {
      "x":"2015-12-01T03:25:25.657Z",
      "y":160
   },
   {
      "x":"2015-11-30T03:25:25.657Z",
      "y":137
   }
]
```

Javascript version:

```js
const data = { thirtyDays: [{"x":"2015-12-29T03:25:25.656Z","y":135},{"x":"2015-12-28T03:25:25.657Z","y":170},{"x":"2015-12-27T03:25:25.657Z","y":113},{"x":"2015-12-26T03:25:25.657Z","y":158},{"x":"2015-12-25T03:25:25.657Z","y":169},{"x":"2015-12-24T03:25:25.657Z","y":189},{"x":"2015-12-23T03:25:25.657Z","y":151},{"x":"2015-12-22T03:25:25.657Z","y":124},{"x":"2015-12-21T03:25:25.657Z","y":140},{"x":"2015-12-20T03:25:25.657Z","y":173},{"x":"2015-12-19T03:25:25.657Z","y":175},{"x":"2015-12-18T03:25:25.657Z","y":130},{"x":"2015-12-17T03:25:25.657Z","y":166},{"x":"2015-12-16T03:25:25.657Z","y":110},{"x":"2015-12-15T03:25:25.657Z","y":115},{"x":"2015-12-14T03:25:25.657Z","y":153},{"x":"2015-12-13T03:25:25.657Z","y":182},{"x":"2015-12-12T03:25:25.657Z","y":144},{"x":"2015-12-11T03:25:25.657Z","y":184},{"x":"2015-12-10T03:25:25.657Z","y":118},{"x":"2015-12-09T03:25:25.657Z","y":188},{"x":"2015-12-08T03:25:25.657Z","y":161},{"x":"2015-12-07T03:25:25.657Z","y":158},{"x":"2015-12-06T03:25:25.657Z","y":132},{"x":"2015-12-05T03:25:25.657Z","y":109},{"x":"2015-12-04T03:25:25.657Z","y":152},{"x":"2015-12-03T03:25:25.657Z","y":197},{"x":"2015-12-02T03:25:25.657Z","y":143},{"x":"2015-12-01T03:25:25.657Z","y":160},{"x":"2015-11-30T03:25:25.657Z","y":137}].map((data) => { data.x = new Date(data.x); return data;})
}
```

## Demo

https://jsfiddle.net/austinpray/rj7p4kx5/

This should net you the following in the latest chrome:

```
Error: Invalid value for <path> attribute d="M NaN, 295.18716577540107 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 158.7121212121212 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 380.9714795008913 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 205.50356506238862 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 162.6114081996435 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 84.62566844919786 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 232.79857397504458 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 338.0793226381462 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 275.69073083778966 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 147.01426024955435 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 139.21568627450978 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 314.6836007130125 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 174.3092691622103 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 392.6693404634581 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 373.1729055258467 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 225 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 111.92067736185385 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 260.09358288770056 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 104.12210338680921 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 361.4750445632799 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 88.52495543672012 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 193.80570409982175 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 205.50356506238862 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 306.8850267379679 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 396.5686274509804 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 228.8992869875223 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 53.43137254901961 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 263.9928698752228 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 197.704991087344 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="M NaN, 287.3885918003565 m -3, 0\n      a 3, 3 0 1,0 6,0\n      a 3, 3 0 1,0 -6,0"setInnerHTML @ react.js:16850ReactMount._mountImageIntoNode @ react.js:11413wrapper @ react.js:12346mountComponentIntoNode @ react.js:10861Mixin.perform @ react.js:15623batchedMountComponentIntoNode @ react.js:10875Mixin.perform @ react.js:15623ReactDefaultBatchingStrategy.batchedUpdates @ react.js:8455batchedUpdates @ react.js:13699ReactMount._renderNewRootComponent @ react.js:11069wrapper @ react.js:12346ReactMount._renderSubtreeIntoContainer @ react.js:11143ReactMount.render @ react.js:11163wrapper @ react.js:12346(anonymous function) @ embedded:13transform.run @ babel.js:3192exec @ babel.js:3247runScripts @ babel.js:3284
react.js:16850 Error: Invalid value for <path> attribute d="MNaN,287.3885918003565LNaN,197.704991087344LNaN,263.9928698752228LNaN,53.43137254901961LNaN,228.8992869875223LNaN,396.5686274509804LNaN,306.8850267379679LNaN,205.50356506238862LNaN,193.80570409982175LNaN,88.52495543672012LNaN,361.4750445632799LNaN,104.12210338680921LNaN,260.09358288770056LNaN,111.92067736185385LNaN,225LNaN,373.1729055258467LNaN,392.6693404634581LNaN,174.3092691622103LNaN,314.6836007130125LNaN,139.21568627450978LNaN,147.01426024955435LNaN,275.69073083778966LNaN,338.0793226381462LNaN,232.79857397504458LNaN,84.62566844919786LNaN,162.6114081996435LNaN,205.50356506238862LNaN,380.9714795008913LNaN,158.7121212121212LNaN,295.18716577540107"
```

**Comment by mdramos** [comment]
+1

**Comment by mdramos** [comment]
I have a temporary fix. Convert the date to milliseconds: like so  '''+(new Date(data.x))'''

**Comment by boygirl** [comment]
The fix @mdramos made to victory-chart has also not yet been released. https://github.com/FormidableLabs/victory-chart/pull/80. I'll release a patch that includes that fix and see if it helps

**Comment by exogen** [comment]
I can get the chart to work by manually setting `tickValues` and `tickFormat` on `VictoryAxis`, like so: https://jsfiddle.net/rj7p4kx5/12/

But that shouldn't be required of course – curious to see if ^ that fix changes anything.

**Comment by boygirl** [comment]
hopefully. I haven't been able to reproduce the problem locally when importing victory-chart

**Comment by mdramos** [comment]
Ah I'll test my original fix and see if that was the issue. Will do so soon.

**Comment by exogen** [comment]
Found #139 while testing – Date related but not an issue here.

**Comment by mdramos** [comment]
Solved: This happens again because of date inconsistency styles. Not sure what changed or why the chart now wants milliseconds, but using primitive values when creating the domain solves date inconsistency issues. The fix is here: FormidableLabs/victory-chart#80

@exogen your code works because domain isn't being calculated by VictoryChart behind the scene when you manually provide the values yourself (or at least thats what I think is happening)
